oak_functions/sdk/oak_functions/src/lib.rs

```diff
@@ -76,6 +76,19 @@ pub fn storage_get_item(key: &[u8]) -> Result<Option<Vec<u8>>, OakStatus> {
     }
 }
 
+/// Writes a debug log message.
+///
+/// These log messages are considered sensitive, so will only be logged by the runtime if the
+/// `oak_unsafe` feature is enabled.
+///
+/// See [`write_log_message`](https://github.com/project-oak/oak/blob/main/docs/oak_functions_abi.md#write_log_message).
+pub fn log(message: &str) -> Result<(), OakStatus> {
+    let message = &format!("[Wasm] {}", message);
```

Comment:
I don't think this should be done by the SDK, it belongs to the logging logic in the runtime.

Reply:
Moved.